### PR TITLE
[IMP] Add README files to base_phone and base_phone_popup modules

### PR DESCRIPTION
**Pull request by foutoucour**
Addition of README files to ease the reading for their descriptions

### CODE REVIEW

File: base_phone_popup/__openerp__.py [hunk 1]
```diff
@@ -26,38 +26,10 @@
...
-TODO : document this new feature on the Akretion Web site :
-http://www.akretion.com/products-and-services/openerp-asterisk-voip-connector
-""",
     'author': "Akretion,Odoo Community Association (OCA)",
     'website': 'http://www.akretion.com/',
     'depends': ['base_phone', 'web_action_request'],
-    'data': [
-        'res_users_view.xml',
-        ],
+    'data': ['res_users_view.xml',],
```
Comment (yvaucher) [inline base_phone_popup/__openerp__.py]:
```
./base_phone_popup/__openerp__.py:32:34: E231 missing whitespace after ','
```

Please remove the `,` before `]`

**Comment by yvaucher** [comment]
Just a tiny PEP8 issue

Otherwise :+1: 

Thanks

File: base_phone/README.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+============
+
+To install this module, you need to:
+
+* to install phonenumbers: pip install phonenumbers
+
+Usage
+=====
+
+Please contact Alexis de Lattre from Akretion <alexis.delattre@akretion.com>
```
Comment (pedrobaeza) [inline base_phone/README.rst]: This shouldn't be here. Please remove entire section.

File: base_phone_popup/README.rst [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+=================
+
+When the user receives a phone call, OpenERP can automatically open the
+corresponding partner/lead/employee/... in a pop-up without any action from the
+user.
+
+Installation
+============
+
+The module *web_action_request* can be downloaded with Mercurial:
```
Comment (pedrobaeza) [inline base_phone_popup/README.rst]: Put: `It depends on modules x, y and z...`

**Comment by foutoucour** [comment]
The fail seems out of scope: _2015-04-10 19:07:52,549 8050 CRITICAL openerp_template openerp.service.server: Failed to initialize database `openerp_template`._

Do you agree?

**Comment by alexis-via** [comment]
:+1: 
It seems we need a rebase

**Pull request closed without merging**